.github/workflows/terraform.yml

```diff
@@ -6,9 +6,13 @@ on:
       - main
     paths:
       - 'infra/tf-next/**'
+  # question: do we need the PR trigger for terraform? maybe just
+  # config check for PRs and the terraform apply only runs for
+  # merges to main?
```

Comment:
Oh I thought from this it would run on both PRs and merges to main. Like every commit in a PR will run this because we have the trigger on both the push and the PR (I think):
```
on:
  push:
    branches:
      - main
    paths:
      - 'infra/tf-next/**'
  pull_request:
    paths:
      - 'infra/tf-next/**'
    types: [ready_for_review]
```

Reply:
https://github.com/fleek-network/ursa/blob/cc6566dfd32d60b0fca257ea9c6725cd62d721d3/.github/workflows/terraform.yml#L87
This line prevents the actual apply from running unless it's on main